backend/game/engine/player_queue.py

```diff
@@ -0,0 +1,45 @@
+import itertools
+
+
+class GamePlayers():
+
+    def __init__(self):
+        self.all_players = []
+        self.dead_players = []
+        self.alive_players = []
+        self.current_player = None
+
+    def _add_initial_player(self, player):
+        self.all_players.append(player)
+
+    def _remove_initial_player(self, player):
+        if player in self.all_players:
+            self.all_players.remove(player)
+
+    def set_player_order(self):
+        # TODO: set this based off of dice roll minigame
+        # Currently just the order in which they were added to the list
+        self.alive_players = self.all_players
+        self.player_cycle = itertools.cycle(self.all_players)
+
+    def get_current_player(self):
+        if not self.current_player:
+            self.get_next_player()
+        return self.current_player
+
+    def get_next_player(self):
+        self.current_player = self._cycle_next_alive_player()
+
+    def get_alive_player(self):
+        return self.alive_players
```

Comment:
As stated earlier, this should probably look through all players and simply recheck whether they're alive. A helper `_is_alive` method would probably be best.